app/src/main/res/layout/dialog_fragment_parking_spaces_setting.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <TextView
+        android:id="@+id/text_dialog_fragment_parking_spaces_setting"
+        style="@style/TitleStyleDialogFragmentSpacesSetting"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent" />
+    <EditText
+        android:id="@+id/number_input_dialog_fragment_parking_spaces_setting"
+        style="@style/InputStyleDialogFragmentSpacesSetting"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="@id/text_dialog_fragment_parking_spaces_setting"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        android:autofillHints="false"
+        android:inputType="number"
```

Comment:
you can add these two properties to the dialog 
android:autofillHints="false"
android:inputType="number"

Reply:
done, except for android:autofillHints="false" needs min API 26 and i´m using currently API 23. Left it where it was.